src/app/OperationalSessionSetup.cpp

```diff
@@ -291,6 +295,21 @@ CHIP_ERROR OperationalSessionSetup::EstablishConnection(const ReliableMessagePro
     mCASEClient = mClientPool->Allocate();
     ReturnErrorCodeIf(mCASEClient == nullptr, CHIP_ERROR_NO_MEMORY);
 
+    // TODO: Combine LargePayload flag with DNS-SD advertisements from peer
```

Comment:
Create a ticket to track this task

Reply:
https://github.com/project-chip/connectedhomeip/issues/29539 should be usable for this.